legal-api/report-templates/amalgamationApplication.html

```diff
@@ -1,13 +1,48 @@
 [[macros.html]]
-
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-      <title>Amalgamation Application</title>
-      <meta charset="UTF-8">
-      <meta name="author" content="BC Registries and Online Services">
-      [[common/style.html]]
-    </head>
-    <body>
-    </body>
+  <head>
+    <title>Amalgamation Application</title>
+    <meta charset="UTF-8">
+    <meta name="author" content="BC Registries and Online Services">
+    [[common/style.html]]
+  </head>
+  <body>
+    <div class="header">
+      <table class="header-table" role="presentation">
+        <tr class="no-page-break">
+          <td>
+            [[logo.html]]
+          </td>
+          <td>
+            <div class="report-type">AMALGAMATION APPLICATION</div>
+            <div class="report-type-desc">{{ entityDescription }} - {{ entityAct }}</div>
+          </td>
+        </tr>
+      </table>
+      <div class="business-name-header">
+        <label class="lbl-business-name">{{nameRequest.legalName}}</label>
+      </div>
+    </div>
+    [[common/businessDetails.html]]
+    <div class="no-page-break">
+      <div class="section-header mt-5 pt-4">AMALGAMATION APPLICATION</div>
+    </div>
+    [[amalgamation/nameRequest.html]]
+    [[amalgamation/effectiveDate.html]]
+    [[amalgamation/approvalType.html]]
+    [[amalgamation/amalgamatingCorp.html]]
+    [[amalgamation/amalgamationStmt.html]]
+    [[amalgamation/completingParty.html]]
+    [[amalgamation/completingPartyStmt.html]]
+
+    <div class="section-header mt-7 pt-4">NOTICE OF ARTICLES</div>
+      <div class="container pt-4">
+        [[common/statement.html]]
+        [[incorporation-application/benefitCompanyStmt.html]]
```

Comment:
Seems like there is at least once difference in the amalgamated version of the statement.

![image](https://github.com/bcgov/lear/assets/6685223/cd4dcd9f-0a80-4192-99a4-037450cf8e64)

Maybe you should create a snippet specific to amalgamations under `/template-parts/amalgamation`